README.md

```diff
@@ -1,5 +1,7 @@
 # CSI Proxy
 
+> Important: CSI Proxy is planning to migrate to v2, where active development is done on the [`library-development`](https://github.com/kubernetes-csi/csi-proxy/tree/library-development) branch. When the KEP for the v2 migration is approved, it will become the `master` branch. The current v1 CSI Proxy is going to be preserved long term on the [`v1.x`](https://github.com/kubernetes-csi/csi-proxy/tree/v1.x) branch.
```

Comment:
```suggestion
> There's a KEP about moving CSI Drivers to HostProcess containers in https://github.com/kubernetes/enhancements/issues/3636 which involves making CSI Proxy a Go library, development for this work is happening in the `library-development` branch
```

The v1.x, master, library-development strategy will be part of the KEP

Reply:
Changed